src/server/status_server.rs

```diff
@@ -240,6 +339,7 @@ impl StatusServer {
                             (Method::GET, "/status") => Box::new(ok(Response::default())),
                             (Method::GET, "/pprof/profile") => Self::dump_prof_to_resp(req),
```

Comment:
I also think we can change the path to `/debug/pprof/heap`, the same as Go sytle.